tests/hash/test_parse_checksum.py

```diff
@@ -0,0 +1,168 @@
+       # -p linux64 -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'linux64',
+               'version': '52.0.2'},
+      'url': 'https://archive.mozilla.org/pub/firefox/releases/52.0.2/SHA512SUMS',
+      'hash': 'f84b186c83e7cc3cc2bcab136029e299cfb4cbf89891f07a7f0c79584df18c91' \
+              '4c24c51615e6bdb677571e194e964cd6d49cdf10f76f68f3b7b9bcaae50ceb3e'},
+       # -p linux64 -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'mac',
+               'version': '52.0.2'},
+      'url': 'https://archive.mozilla.org/pub/firefox/releases/52.0.2/SHA512SUMS',
+      'hash': '51bbac18a401b12dbb0a7e52519de6631cd217e312682d01d10a202b52e029be' \
+              '71f601cb026866b78f1c76e6176c4197982df2657e61f77a789283b41ad0da1b'},
+]
+
+tests_candidate_scraper = [
+    # -p win32 -v latest
+    {'args': {'application': 'firefox',
+              'platform': 'win32',
+              'version': '53.0b9'},
+     'url': 'https://archive.mozilla.org/pub/firefox/candidates/53.0b9-candidates/build1/SHA512SUMS',
+     'hash': 'c0be3e32850ee3cfe4c6d4bee112df8488da248b907b801e7a33eef0dfd0385f0' \
+             'cf1031ee39c2dda51c10661c0d822e70bdf7fb86d23a2af8464d2594a4f7fbc'},
+     # -p win64 -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'win64',
+               'version': '53.0b9'},
+      'url': 'https://archive.mozilla.org/pub/firefox/candidates/53.0b9-candidates/build1/SHA512SUMS',
+      'hash': '215095f65bf9fc5029cb2d1befce91a54283263552281dad92b34dbcbb8a40e2' \
+              '2ebc36501b34827906a2da44d0981ae2652b101af21df6d59f7880ecb1b238b4'},
+      # -p linux -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'linux',
+               'version': '53.0b9'},
+      'url': 'https://archive.mozilla.org/pub/firefox/candidates/53.0b9-candidates/build1/SHA512SUMS',
+      'hash': '3a32f608aa3252db784d50a1beec2654e1ebd31e7985446d312d8df8a809303e' \
+              '4581bcad8572aa37a91d4d09ded0defe4290c528142dbe2d9f820d0261bf17a3'},
+       # -p linux64 -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'linux64',
+               'version': '53.0b9'},
+      'url': 'https://archive.mozilla.org/pub/firefox/candidates/53.0b9-candidates/build1/SHA512SUMS',
+      'hash': 'a71d5b34a58b7c440711f838e268a17a7a515d6f641c1c29e1fde5e5313aab73' \
+              '2dfc38970e40f3cd209bbb32ad1618358fa5ebba77b0e6d88c2a44c46f630cc0'},
+       # -p linux64 -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'mac',
+               'version': '53.0b9'},
+      'url': 'https://archive.mozilla.org/pub/firefox/candidates/53.0b9-candidates/build1/SHA512SUMS',
+      'hash': 'ec13593e7ba46c4f86ab15b888c33346a05efe0144120f8ac7d65633b5d7e259' \
+              '6e317bf073647cf04db400ae0f5d97411a24bf1f0a9691f49ca51ac995f339d6'},
+]
+
+tests_daily_scraper = [
```

Comment:
If those tests should cover the daily scraper then you have to update the parameters for calling mozdownload.